### PR TITLE
Create an AWS commons extension

### LINKED ISSUE #566: Create an AWS common extension
Containing the shared part among the AWS extensions.

### PR DESCRIPTION
**Pull request by oscerd**
Fixes #566

### CODE REVIEW

File: extensions/aws-commons/runtime/src/main/resources/META-INF/quarkus-extension.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+name: "Camel Quarkus AWS Commons"
+description: "A Camel Amazon Commons Component"
+metadata:
+  keywords:
```
Comment (ppalaga) [inline extensions/aws-commons/runtime/src/main/resources/META-INF/quarkus-extension.yaml]:
```suggestion
  unlisted: true
  keywords:
```

File: tooling/package-maven-plugin/src/main/java/org/apache/camel/quarkus/maven/PrepareCatalogQuarkusMojo.java [hunk 2]
```diff
@@ -215,6 +215,7 @@ protected void appendOthers(List<CamelQuarkusExtension> extensions, CamelCatalog
                 final String title = ext.getName().orElseThrow(() -> new MojoExecutionException(
                         "name is missing in " + ext.getRuntimePomXmlPath()));
                 model.put("title", title);
+                System.err.println("title " + title + " and desc " + ext.getDescription());
```
Comment (ppalaga) [inline tooling/package-maven-plugin/src/main/java/org/apache/camel/quarkus/maven/PrepareCatalogQuarkusMojo.java]: Is this a leftover maybe?

Comment (oscerd) [reply tooling/package-maven-plugin/src/main/java/org/apache/camel/quarkus/maven/PrepareCatalogQuarkusMojo.java]: Yes, my fault :-)

**Comment by oscerd** [comment]
ok to test

**Comment by oscerd** [comment]
updated

**Comment by ppalaga** [comment]
> I think the right place for this kind of ancillary extension would be `extensions/support/aws` and the artifactId should match the existing conventions - something like `camel-quarkus-support-aws`

Is there any reason not to move the extesion to `extensions/support/aws`?

**Comment by oscerd** [comment]
This was the first idea. It could be moved for sure.

**Comment by oscerd** [comment]
Btw I'll move the extension when I can.

**Comment by oscerd** [comment]
ok to test

**Comment by lburgazzoli** [comment]
ok to test

**Comment by oscerd** [comment]
Ok to test

**Pull request merged**